core/src/main/scala/akka/persistence/jdbc/query/JournalSequenceActor.scala

```diff
@@ -38,15 +38,15 @@ object JournalSequenceActor {
    * Efficient representation of missing elements using NumericRanges.
    * It can be seen as a collection of OrderingIds
    */
-  private case class MissingElements(elements: Seq[NumericRange[OrderingId]]) {
+  case class MissingElements(elements: Seq[NumericRange[OrderingId]]) {
     def addRange(from: OrderingId, until: OrderingId): MissingElements = {
       val newRange = from.until(until)
       MissingElements(elements :+ newRange)
     }
     def contains(id: OrderingId): Boolean = elements.exists(_.containsTyped(id))
     def isEmpty: Boolean = elements.forall(_.isEmpty)
   }
-  private object MissingElements {
+  object MissingElements {
```

Comment:
Make the multi-param `JournalSequenceActor.receive(currentMaxOrdering ...)` private instead of changing these (it's an internal factory for `Receive ` instances, not called from the outside)